interactions/context.py

```diff
@@ -17,7 +17,7 @@
 log: Logger = get_logger("context")
 
 
-class Context(DictSerializerMixin):
+class ContextMixin(DictSerializerMixin):
```

Comment:
```suggestion
class _Context(DictSerializerMixin):
```

The naming for mixins will be changing since we are considering dunder to be applied to our attributes, instances and methods as of the 2022 roadmap.